regression/pages/studio/pages_page_studio.py

```diff
@@ -144,23 +144,21 @@ def click_and_verify_see_an_example(self):
             'img[alt="Preview of Pages in your course"]', 'Pop up visibility'
         )
 
-    def click_hide_show_toggle(self, index=0):
+    def click_hide_show_toggle(self):
```

Comment:
nit: should probably rename this method. Something like toggle_wiki_page_display.
Oh, and update the docstring also.